### PR TITLE
The sending to the DAO was changed the id of the member to delete and…

### PR DESCRIPTION
**Pull request by AlanAlexis11**
**Summary**

- Delete duplicate migration role
- The id of the member to be deleted is sent
- The generate JWT with USER, unstructured from req
- The secret value was removed
- The use of the token in the endpoints was commented

### CODE REVIEW

File: controllers/members/dao.js [hunk 1]
```diff
@@ -5,15 +5,13 @@ module.exports = {
         await db.Members.create(member);
     },
     async deleteMemberDAO(memberID){
-        console.log("eee")
         return await db.Members.destroy({
             where:{id:memberID}
```
Comment (guillote87) [inline controllers/members/dao.js:9]: revisar el memberId viaja como objeto

**Review by guillote87 — APPROVED** [review]
Revisar el deletemember

File: app.js [hunk 2]
```diff
@@ -3,13 +3,18 @@ const express = require('express');
...
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
-const cors = require('cors')
+const cors = require('cors');
 require('dotenv').config()
 
 const indexRouter = require('./routes/index');
 const usersRouter = require('./routes/users');
 const membersRouter = require('./routes/members');
+<<<<<<< HEAD
```
Comment (JonaPortillo) [inline app.js]: Eliminar restos del merge

**Review by JonaPortillo — APPROVED** [review]
ok

**Pull request merged**